blockchainetl/exporters.py

```diff
@@ -119,9 +119,16 @@ def serialize_field(self, field, name, value):
         return serializer(value)
 
     def _join_if_needed(self, value):
+        def to_string(x):
+            if isinstance(x, dict):
+                # Separators without whitespace for compact format.
+                return JSONEncoder(separators=(',', ':')).encode(x)
```

Comment:
I'm not 100% convinced this is definitely the best format for a stringified `List[Dict]` but okay. My first thought was just to `str(my_list_of_dicts)` (typically returns single quotes), or `json.dumps(my_list_of_dicts)` (double quotes), or the other one I've seen is using `json.dumps(json.dumps(my_list_of_dicts))` (escaped double quotes). But as far as I'm concerned, it's no biggie.

Reply:
@charlielewisme Agree it's definitely not the best format (see description as well). My concern was that switching to JSON serialisation for all lists might break backwards compatibility elsewhere. E.g. for logs we would change the format of [topics](https://github.com/blockchain-etl/ethereum-etl/blob/develop/ethereumetl/domain/receipt_log.py#L33) from `"topic1,topic2"` to `"[""topic1"",""topic2""]"`. Relying on the string representation of Python objects also seemed less than ideal. But agree, the current solution is a compromise that might not offer any benefit either. Open to other suggestions:)

Reply:
Agree with not breaking backwards compatibility as high priority. I'm not familiar with established use cases for this cli that rely on csv so, but for me this is a decent enough compromise.